### PR TITLE
[IA-4003] add reminder text to modal

### PR DESCRIPTION
**Pull request by ghost**
reminder text added to delete runtime modal with a stylized box.
Here's Azure:

https://user-images.githubusercontent.com/123021647/215855835-d322b740-3820-440b-a315-5a1d04a28f29.mov

And here's Google:

https://user-images.githubusercontent.com/123021647/215855847-0db4d72b-9c24-4563-9a43-45a408b9ad31.mov

### CODE REVIEW

**Comment by cahrens** [comment]
I don't understand this message -- a persistent disk is already created for a Jupyter environment on Azure (see my screenshot where I have one running). But if there wasn't a persistent disk, then why include the checkbox under the message?

Also, the spacing is a bit odd around the bluish box. And check for color contrast.

image

**Comment by ghost** [comment]
We're in a tricky middle phase where we don't yet support PDs for Azure VMs (but will be coming relatively soon). We're looking to implement a temporary solution to inform users that selecting the checkbox does nothing. Once PDs are supported, then we'll revert this change. 

Good call out on the Cloud Env Page having the specific title of Persistent Disk. It's misleading for Azure VMs, where they have a local storage disk but it's not detachable/reattach-able (aka PDs). 

Another option is to grey out the checkbox for Azure VMs only. It won't solve the "persistent disk" name issue, but given that it's a short test period it'll hopefully be of minimal impact.

**Comment by cahrens** [comment]
Why not just hide the checkbox completely then? It seems weird to put in messaging about why something is disabled/doesn't work when the user doesn't yet have a concept that they should expect it to be there (I'm thinking here that our Azure preview users are mostly folks who are new to Terra).

**Comment by ghost** [comment]
Because the functionality is available and useful for GCP VMs

**Comment by ghost** [comment]
Or are you suggesting having a different pop up/modal when selecting Azure VMs?

**Comment by cahrens** [comment]
Right, I meant don't include the checkbox for Azure cloud environments-- it definitely should stay for GCP cloud environments.

It is my understanding that the preview users will only be on Azure, so it's not like they will see the checkbox for their Google cloud environments and wonder why it isn't present for their Azure cloud environments.

**Comment by jdcanas** [comment]
Yeah, I'm, not the biggest fan of this message wording for the same reason @cahrens mentions. I think it should say something like `Deleting a runtime requires deleting a persistent disk for azure`. We do support PDs, just not management of them outside the VM lifecycle yet

**Comment by ghost** [comment]
I'd rather keep this simple. I have two options:

1. We simply don't show any additional text. The body of the message already informs the user that deleting the VM will delete your storage, and if you want to keep your files then you move them to workspace storage.

2. We include additional text that reads "Deleting your Virtual Machine will also delete the attached persistent disk." It's simple and to the point. 

With the assumption that the message for #2 only pops up when you select an Azure VM, not a GCP VM. Thoughts @cahrens @jdcanas ?

File: src/pages/Environments.js [hunk 1]
```diff
@@ -666,7 +682,7 @@ export const Environments = ({ nav = undefined }) => {
               }
             },
             {
-              size: { min: '10em', grow: 0 },
+              size: { min: '11em', grow: 0 },
```
Comment (n8landolt) [inline src/pages/Environments.js:685]: I suggested this edit while in the file. Prior to this edit, hovering Last Accessed would wrap the d to the next line due to the sort arrow.

**Review by n8landolt — APPROVED** [review]

**Review by lucymcnatt — APPROVED** [review]

**Pull request merged**